Mathlib/Algebra/Order/Archimedean/Class.lean

```diff
@@ -452,3 +452,127 @@ theorem liftOrderHom_mk (f : M → α) (h : ∀ a b, mk a ≤ mk b → f a ≤ f
 end LiftHom
 
 end MulArchimedeanClass
+
+variable (M) in
+/-- Subtype of `MulArchimedeanClass` that removes the top element. -/
+@[to_additive ArchimedeanClass₀ "Subtype of `ArchimedeanClass` that removes the top element."]
+abbrev MulArchimedeanClass₁ := {A : MulArchimedeanClass M // A ≠ ⊤}
```

Comment:
I still dislike this name. What about
```suggestion
abbrev FiniteMulArchimedeanClass := {A : MulArchimedeanClass M // A ≠ ⊤}
```
or alike?

Reply:
I don't hate it. It just take me a while to realize "Finite" means the opposite of infinite (⊤). I'll change it to this if I don't come up with a better one